src/frontends/tensorflow_common/src/op/div.cpp

```diff
@@ -0,0 +1,29 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "common_op_table.hpp"
+#include "openvino/op/divide.hpp"
+
+using namespace std;
+using namespace ov::op;
+
+namespace ov {
+namespace frontend {
+namespace tensorflow {
+namespace op {
+OutputVector translate_div_op(const NodeContext& node) {
```

Comment:
We have a template translator for simple binary ops: https://github.com/openvinotoolkit/openvino/blob/6060683f4c1ac320f1c01fc5c8fc8b8f53f93f02/src/frontends/tensorflow_common/src/op/binary_op.cpp#L63

Let's reuse it for this operation.

Reply:
Could you please show me an example of using this template, if available?

Reply:
You can use `Add` or other operations from this list as example. It should be enough to add a line with `Div` in the list:
https://github.com/openvinotoolkit/openvino/blob/139242d47d6ec8bdb42a05954f51381968435e53/src/frontends/tensorflow_common/src/op/binary_op.cpp#L63

Here is implementation of `translate_binary_op` template: 
https://github.com/openvinotoolkit/openvino/blob/139242d47d6ec8bdb42a05954f51381968435e53/src/frontends/tensorflow_common/src/op/binary_op.cpp#L39

As `Div` translator has no `Div`-specific code I recommend to reuse the template.